### ISSUE TITLE
TypeError: _flatten() takes exactly 2 arguments (1 given)

### ISSUE THREAD

**Issue by skynet** [ansible/ansible#12485]
With `ansible 2.0.0 (devel c83f51b7f2) last updated 2015/09/23` and a playbook like this:

```
- name: Build NGINX Config
  template: src=templates/vhost.j2 dest={{item.name}}.conf
  with_items:
  - "{{ properties }}"
```

I get this error:

```
An exception occurred during task execution. The full traceback is:
Traceback (most recent call last):
  File "/usr/local/src/ansible/lib/ansible/executor/process/worker.py", line 122, in run
    executor_result = TaskExecutor(host, task, job_vars, new_play_context, self._new_stdin, self._loader, shared_loader_obj).run()
  File "/usr/local/src/ansible/lib/ansible/executor/task_executor.py", line 89, in run
    items = self._get_loop_items()
  File "/usr/local/src/ansible/lib/ansible/executor/task_executor.py", line 179, in _get_loop_items
    items = self._shared_loader_obj.lookup_loader.get(self._task.loop, loader=self._loader, templar=templar).run(terms=loop_terms, variables=vars_copy)
  File "/usr/local/src/ansible/lib/ansible/plugins/lookup/items.py", line 26, in run
    return self._flatten(terms)
TypeError: _flatten() takes exactly 2 arguments (1 given)

fatal: [localhost]: FAILED! => {"failed": true, "stdout": ""}
```

**Comment by skynet** [comment]
This was fixed after I reset the $PYTHONPATH to /usr/lib/python2.6

**Comment by skynet** [comment]
Actually, it isn't fixed, the reset just brought me to a stable version and that's why I got the wrong idea that it works now.

**Comment by jimi-c** [comment]
Just fixed this in devel (cbbb27076152dd1a3bbce58f7fdba58237672896 and 65630d2ce16e39f724a7f3fbdf6bef91afc81cfb), so should be ok now.
